crates/chia-consensus/src/spendbundle_conditions.rs

```diff
@@ -108,6 +119,34 @@ mod tests {
             .fold(0, |sum, spend| sum + spend.create_coin.len());
         assert_eq!(create_coins, additions);
         assert_eq!(conditions.cost, cost);
+        // Generate a block with the same spend bundle and compare its cost
+        let program_spends = bundle.coin_spends.iter().map(|coin_spend| {
+            (
+                coin_spend.coin,
+                &coin_spend.puzzle_reveal,
+                &coin_spend.solution,
+            )
+        });
+        let program = solution_generator(program_spends).expect("solution_generator failed");
+        let blocks: &[&[u8]] = &[];
+        let block_conds = run_block_generator2::<_, MempoolVisitor, _>(
+            &mut a,
+            program.as_slice(),
+            blocks,
+            11_000_000_000,
+            MEMPOOL_MODE,
+            &TEST_CONSTANTS,
+        )
+        .expect("run_block_generator2 failed");
+        // The cost difference here is because get_conditions_from_spendbundle
+        // uses solution_generator which wraps the program in a quote, and we
+        // don't pay the size cost nor execution cost for that there.
```

Comment:
this isn't quite right. the quote difference here isn't *caused* by an implementation detail. It's deliberate and, ideally, the comment would explain why it's deliberate. Specifically, `get_conditions_from_spendbundle()` does not include the overhead to make a block.

Reply:
Done.